frontend/src/pages/Curator/Curator.jsx

```diff
@@ -41,6 +42,8 @@ const Curator = () => {
       tryItMode={false}
       prompt={prompt}
       setPrompt={setPrompt}
+      synopsis={synopsis}
+      setSynopsis={setSynopsis}
```

Comment:
Also add state & pass state props to the TryItCurator

Reply:
After this, LGTM

Reply:
resolvedddd